### PR TITLE
docs: document the recommended setup

### PR DESCRIPTION
**Pull request by merlinnot**
This Pull Request fulfills the following requirements:

- [x] The commit message follows our guidelines.
- [x] Tests for the changes have been added if needed.
- [x] Does not affect documentation or it has been added or updated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,12 +16,26 @@ require branches to be up to date, require status checks to pass).
 
 ## Usage
 
-The Action supports two run triggers:
+The Action supports three run triggers:
```
Comment (JrSchild) [inline README.md:19]: [four](https://github.com/ridedott/merge-me-action/blob/master/src/index.ts#L32)

**Review by JrSchild — APPROVED** [review]
LGTM 👍

File: README.md (hunk 1 above)
Comment (merlinnot) [reply README.md:19]: But how?... I mean, even `GITHUB_TOKEN` is read-only in `pull_request` trigger, right?

**Review by viestat — APPROVED** [review]
Documentation is clear :D

File: README.md [hunk 2]
```diff
@@ -81,6 +95,86 @@ jobs:
...
+        run: npm run test
+  merge-me:
+    name: Merge me!
+    needs:
+      # List all required job names here.
+      - format
+      - lint
+      - spellcheck
+      - test
+      - types
```
Comment (mad-it) [inline README.md:158]:
```suggestion
      - test
```

**Pull request merged**